common/djangoapps/third_party_auth/provider.py

```diff
@@ -26,6 +26,10 @@ def _enabled_providers(cls):
                 provider = SAMLProviderConfig.current(idp_slug)
                 if provider.enabled and provider.backend_name in _PSA_SAML_BACKENDS:
                     yield provider
+        for consumer_key in LTIProviderConfig.key_values('lti_consumer_key', flat=True):
```

Comment:
I assume this only makes sense if the `ENABLE_LTI_PROVIDER` feature flag is on? If so, please add a check for that here so we don't bother querying the database for LTI providers when that functionality is not even enabled. (Similar to what is done above for SAML providers)